### PR TITLE
Update de.js

### PR DESCRIPTION
**Pull request by arbadacarbaYK**

### CODE REVIEW

**Comment by callebtc** [comment]
Nice! I think we need a last `make format` for the checks to pass!

File: lnbits/static/i18n/de.js [hunk 1]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
   topup_wallet: 'Wallet aufladen',
-  topup_hint: 'Nutze die Wallet-ID um eine beliebige Wallet aufzuladen',
-  restart_tooltip: 'Starte den Server neu um die Änderungen zu übernehmen',
+  topup_hint: 'Nutze die Wallet-ID, um eine beliebige Wallet aufzuladen',
+  restart_tooltip: 'Starte den Server neu, um die Änderungen zu übernehmen',
   add_funds_tooltip: 'Füge Geld zu einer Wallet hinzu.',
   reset_defaults: 'Zurücksetzen',
   reset_defaults_tooltip:
-    'Lösche alle Einstellungen und setze auf die Standardeinstellungen zurück.',
+    'Alle Einstellungen zurücksetzen auf die Standardeinstellungen.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:17]:
```suggestion
  'Alle Einstellungen zurücksetzen auf die Standardeinstellungen.',
```

File: lnbits/static/i18n/de.js [hunk 2]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
-  export_to_phone: 'Exportieren Sie auf das Telefon mit QR-Code',
+    'Einfach zu installieren und kompakt, LNbits kann auf jeder Funding-Quelle im Lightning Netzwerk aufsetzen. Derzeit unterstützt: LND, c-lightning, OpenNode, LNPay und sogar LNbits selbst! Du kannst LNbits für dich selbst betreiben oder anderen die Verwaltung durch dich anbieten. Jede Wallet hat ihre eigenen API-Schlüssel und die Anzahl der Wallets ist unbegrenzt. Die Möglichkeit, Gelder auf verschiedene Accounts mit unterschiedlicher Logik aufteilen zu können macht LNbits zu einem nützlichen Werkzeug für deine Buchhaltung - aber auch als Entwicklungswerkzeug. Erweiterungen bereichern LNbits Accounts um zusätzliche Funktionalität, so dass du mit einer Reihe von neuartigen Technologien auf dem Lightning-Netzwerk experimentieren kannst. Wir haben es so einfach wie möglich gemacht, Erweiterungen zu entwickeln, und als freies und Open-Source-Projekt möchten wir Menschen ermutigen, sich selbst hieran zu versuchen und gemeinsam mit uns neue Funktionalitäten zu entwickeln.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:23]:
```suggestion
  'Einfach zu installieren und kompakt, LNbits kann auf jeder Funding-Quelle im Lightning Netzwerk aufsetzen. Derzeit unterstützt: LND, c-lightning, OpenNode, LNPay und sogar LNbits selbst! Du kannst LNbits für dich selbst betreiben oder anderen die Verwaltung durch dich anbieten. Jede Wallet hat ihre eigenen API-Schlüssel und die Anzahl der Wallets ist unbegrenzt. Die Möglichkeit, Gelder auf verschiedene Accounts mit unterschiedlicher Logik aufteilen zu können macht LNbits zu einem nützlichen Werkzeug für deine Buchhaltung - aber auch als Entwicklungswerkzeug. Erweiterungen bereichern LNbits Accounts um zusätzliche Funktionalität, so dass du mit einer Reihe von neuartigen Technologien auf dem Lightning-Netzwerk experimentieren kannst. Wir haben es so einfach wie möglich gemacht, Erweiterungen zu entwickeln, und als freies und Open-Source-Projekt möchten wir Menschen ermutigen, sich selbst hieran zu versuchen und gemeinsam mit uns neue Funktionalitäten zu entwickeln.',
```

File: lnbits/static/i18n/de.js [hunk 3]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
+  export_to_phone: 'Auf dem Telefon öffnen',
   export_to_phone_desc:
-    'Dieser QR-Code enthält Ihre Wallet-URL mit vollständigem Zugriff. Sie können es von Ihrem Telefon aus scannen, um Ihre Wallet dort zu öffnen.',
+    'Dieser QR-Code beinhaltet vollständige Rechte auf deine Wallet. Du kannste den QR-Code mit Deinem Telefon scannen, um deine Wallet dort zu öffnen.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:26]:
```suggestion
  'Dieser QR-Code beinhaltet vollständige Rechte auf deine Wallet. Du kannste den QR-Code mit Deinem Telefon scannen, um deine Wallet dort zu öffnen.',
```

File: lnbits/static/i18n/de.js [hunk 4]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
   wallets: 'Wallets',
-  add_wallet: 'Füge eine neue Wallet hinzu',
+  add_wallet: 'Wallet hinzufügen',
   delete_wallet: 'Wallet löschen',
   delete_wallet_desc:
-    'Diese ganze Wallet wird gelöscht, die Mittel sind UNWIEDERBRINGLICH.',
+    'Die Wallet wird gelöscht, die hierin beinhalteten Daten hierin oder innerhalb einer Erweiterung sind UNWIEDERBRINGLICH.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:31]:
```suggestion
  'Die Wallet wird gelöscht, die hierin beinhalteten Daten hierin oder innerhalb einer Erweiterung sind UNWIEDERBRINGLICH.',
```

File: lnbits/static/i18n/de.js [hunk 5]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
   donate: 'Spenden',
   view_github: 'Auf GitHub anzeigen',
   voidwallet_active: 'VoidWallet ist aktiv! Zahlungen deaktiviert',
   use_with_caution:
-    'VERWENDEN SIE MIT VORSICHT - %{name} Wallet ist noch in der BETA',
-  toggle_darkmode: 'Dark Mode umschalten',
-  view_swagger_docs: 'LNbits Swagger API-Dokumente anzeigen',
-  api_docs: 'Api docs',
-  commit_version: 'Commit version',
+    'BITTE MIT VORSICHT BENUTZEN- %{name} Wallet ist noch BETA',
```
Comment (dni) [inline lnbits/static/i18n/de.js:39]:
```suggestion
  'BITTE MIT VORSICHT BENUTZEN- %{name} Wallet ist noch BETA',
```

File: lnbits/static/i18n/de.js [hunk 6]
```diff
@@ -9,46 +9,46 @@ window.localisation.de = {
...
   runs_on: 'Läuft auf',
-  credit_hint: 'Drücken Sie Enter, um das Konto zu belasten',
+  credit_hint: 'Klicke Enter, um das Konto zu belasten',
   credit_label: '%{denomination} zu belasten',
   paste_request: 'Anfrage einfügen',
   create_invoice: 'Rechnung erstellen',
   camera_tooltip:
-    'Verwenden Sie die Kamera, um eine Rechnung / QR-Code zu scannen',
-  export_csv: 'Exportieren Sie nach CSV',
+    'Verwenden Sie die Kamera, um eine Rechnung oder einen QR-Code zu scannen',
```
Comment (dni) [inline lnbits/static/i18n/de.js:50]:
```suggestion
  'Verwenden Sie die Kamera, um eine Rechnung oder einen QR-Code zu scannen',
```

File: lnbits/static/i18n/de.js [hunk 7]
```diff
@@ -60,31 +60,31 @@ window.localisation.de = {
...
+  invoice_waiting: 'Rechnung wartend auf Zahlung',
   payment_received: 'Zahlung erhalten',
   payment_sent: 'Zahlung gesendet',
-  outgoing_payment_pending: 'Ausgehende Zahlung ausstehend',
-  drain_funds: 'Leeren Sie die Mittel',
+  outgoing_payment_pending: 'Ausgehende Zahlung wartend',
+  drain_funds: 'Sats abziehen',
   drain_funds_desc:
-    'Dies ist ein LNURL-withdraw QR-Code zum Absaugen aller Mittel aus dieser Brieftasche. Teilen Sie es nicht mit jemandem. Es ist mit balanceCheck und balanceNotify kompatibel, so dass Ihre Brieftasche die Mittel nach dem ersten Abzug kontinuierlich von hier ziehen kann.',
+    'LNURL-withdraw QR-Code, der das Abziehen aller Geldmittel aus dieser Wallet erlaubt. Teile ihn mit niemandem! Kompatibel mit balanceCheck und balanceNotify, so dass deine Brieftasche die Sats nach dem ersten Abzug kontinuierlich von hier abziehen kann.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:72]:
```suggestion
  'LNURL-withdraw QR-Code, der das Abziehen aller Geldmittel aus dieser Wallet erlaubt. Teile ihn mit niemandem! Kompatibel mit balanceCheck und balanceNotify, so dass deine Brieftasche die Sats nach dem ersten Abzug kontinuierlich von hier abziehen kann.',
```

File: lnbits/static/i18n/de.js [hunk 8]
```diff
@@ -60,31 +60,31 @@ window.localisation.de = {
...
   i_understand: 'Ich verstehe',
   copy_wallet_url: 'Wallet-URL kopieren',
   disclaimer_dialog:
-    'Login-Funktionalität wird in einem zukünftigen Update veröffentlicht, für jetzt, stellen Sie sicher, dass Sie diese Seite als Lesezeichen speichern, um zukünftigen Zugriff auf Ihre Wallet zu erhalten! Dieser Service ist in BETA und wir übernehmen keine Verantwortung für Personen, die den Zugriff auf ihre Mittel verlieren.',
-  no_transactions: 'Noch keine Transaktionen',
+    'Login-Funktionalität wird in einem zukünftigen Update veröffentlicht. Bis dahin ist die Speicherung der Wallet-URL als Lesezeichen absolut notwendig, um Zugriff auf die Wallet zu erhalten! Dieser Service ist in BETA und wir übernehmen keine Verantwortung für Verluste durch verlorene Zugriffe.',
```
Comment (dni) [inline lnbits/static/i18n/de.js:76]:
```suggestion
  'Login-Funktionalität wird in einem zukünftigen Update veröffentlicht. Bis dahin ist die Speicherung der Wallet-URL als Lesezeichen absolut notwendig, um Zugriff auf die Wallet zu erhalten! Dieser Service ist in BETA und wir übernehmen keine Verantwortung für Verluste durch verlorene Zugriffe.',
```

**Review by dni — COMMENTED** [review]
this should do the format trick 2 spaces not 4

**Comment by dni** [comment]
we also need the regenerate the bundled files

**Comment by dni** [comment]
closed in favor of https://github.com/lnbits/lnbits/pull/1675

**Pull request closed without merging**